examples/nextjs-typescript-integration/abis/ERC20.json

```diff
@@ -185,4 +185,4 @@
       "type": "function"
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
Can you remove all changes from files in the /examples/ folder from this PR?

I know, they got changed when running `npm run format-all` 
Still, these changes belong to another PR. Thanks